### ISSUE TITLE
Artifacts left at end of OFF animation

### ISSUE THREAD

**Issue by danh32** [mattprecious/swirl#1]
![screenshot_20160311-162237](https://cloud.githubusercontent.com/assets/447085/13719103/e036f272-e7a5-11e5-8ddc-5d504c05197b.png)

These pop up at the very beginning of the ON animation and remain after the end of the OFF animation. I'm seeing this both in my main app, and in the Whorlwind sample app.

It's worth noting that I'm running N preview here, not sure if that could be affecting anything.

**Comment by mattprecious** [comment]
Yeah, I noticed this as well. It's definitely an N-specific issue. I need to do some more testing but I'll probably file an issue for Google.

**Comment by danh32** [comment]
Ah, gotcha! I'll close this out, then.

**Comment by mattprecious** [comment]
Opened http://b.android.com/203586

**Comment by tylerbwong** [comment]
I'm not sure if this is the same kind of issue, or if it's also an issue that should be filed for Google, but I noticed that after the error animation, two small diagonal lines show up on the fingerprint. This screenshot was taken on a device running 8.1, but I saw it when I was running 7.1.2 and 8.0 as well. It shows up for me on both the Robinhood and Credit Karma apps.

![26178256_10208902349956503_1160316592_o](https://user-images.githubusercontent.com/5953895/34433838-cc0c6b5a-ec36-11e7-96f9-03889a5d9cb3.png)
